glusterd2/commands/volumes/volume-create.go

```diff
@@ -260,3 +269,63 @@ func volumeCreateHandler(w http.ResponseWriter, r *http.Request) {
 func createVolumeCreateResp(v *volume.Volinfo) *api.VolumeCreateResp {
 	return (*api.VolumeCreateResp)(createVolumeInfoResp(v))
 }
+
+func checkDisperseParams(req *api.VolCreateReq, v *volume.Volinfo) error {
```

Comment:
@aspandey Can you review this and `getRedundancy` function ?

Reply:
@aspandey Awaiting your review to get this PR merged.